### ISSUE TITLE
Arch needs vulkan-validation-layers to build - add package to README.md

### ISSUE THREAD

**Issue by C0rn3j** [RPCS3/rpcs3#4695]
Otherwise it just fails.
```
[ 86%] Building CXX object rpcs3/CMakeFiles/rpcs3.dir/Emu/RSX/VK/VKFragmentProgram.cpp.o
In file included from /home/c0rn3j/rpcs3/rpcs3/Emu/RSX/VK/VKHelpers.h:15,
                 from /home/c0rn3j/rpcs3/rpcs3/Emu/RSX/VK/VKFormats.h:2,
                 from /home/c0rn3j/rpcs3/rpcs3/Emu/RSX/VK/VKFormats.cpp:2:
/home/c0rn3j/rpcs3/rpcs3/Emu/RSX/VK/VulkanAPI.h:11:10: fatal error: vulkan/vk_sdk_platform.h: No such file or directory
 #include <vulkan/vk_sdk_platform.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [rpcs3/CMakeFiles/rpcs3.dir/build.make:3231: rpcs3/CMakeFiles/rpcs3.dir/Emu/RSX/VK/VKFormats.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /home/c0rn3j/rpcs3/rpcs3/Emu/RSX/VK/VKFragmentProgram.h:5,
                 from /home/c0rn3j/rpcs3/rpcs3/Emu/RSX/VK/VKFragmentProgram.cpp:4:
/home/c0rn3j/rpcs3/rpcs3/Emu/RSX/VK/VulkanAPI.h:11:10: fatal error: vulkan/vk_sdk_platform.h: No such file or directory
 #include <vulkan/vk_sdk_platform.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [rpcs3/CMakeFiles/rpcs3.dir/build.make:3245: rpcs3/CMakeFiles/rpcs3.dir/Emu/RSX/VK/VKFragmentProgram.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:196: rpcs3/CMakeFiles/rpcs3.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

Also while we're at editing README.md, it seems that this sentence is missing a linebreak before it 
"If you have a NVIDIA GPU, you may need to install the libglvnd package."

**Comment by JohnHolmesII** [comment]
This was just removed recently

**Comment by hcorion** [comment]
@C0rn3j is talking about the Arch Linux package called vulkan-validation-layers

**Comment by JohnHolmesII** [comment]
right
it was removed in a pr recently

So it makes sense that there is an issue. I notice that the readme for the arch commands dont have any vulkan sdk stuff listed. Is there no vulkan-dev source for arch?

**Comment by hcorion** [comment]
Arch doesn't have -dev packages, regular packages also include headers and development related things.

**Comment by JohnHolmesII** [comment]
I see. Do you wanna do it or should I? Also, since the readme only lists sudo pacman -S glew openal cmake llvm qt5-base, is that really it? Anything else?

The arch wiki lists two packages as necessary, vilkan-headers and vulkan-validation-layers. Does vvl include the headers or is it only an add-on?

**Comment by hcorion** [comment]
llvm is no longer needed. You can do it, vulkan-validation-layers depends on vulkan-headers, so we only need to mention vvl.

**Comment by hcorion** [comment]
This issue was fixed in 39e3fb46384123b13c453e31618039fd1d5a8d94
